reporting/postman_reporter/report_config.py

```diff
@@ -51,6 +51,11 @@
     "padding": "2rem 1rem",
 }
 
-COLOR_MAP = {"(?)": "darkred", "Pass": "darkgreen", "Fail": "darkred"}
+COLOR_MAP = {
+    "(?)": "darkgoldenrod",
+    "Pass": "darkgreen",
+    "Fail": "darkred",
+    "Partial": "darkgoldenrod",
```

Comment:
hmm.... not sure about adding this.

Reply:
Can you explain why its valuable?

Reply:
It serves to show when a test is partially passing, e.g., when a test case contains multiple assertions and only some of them fail. The other option would be to simply show those as failures.

If you would like to add a change request to modify `darkgoldenrod` to `darkred` it will produce the same output as before.

Reply:
i see, I am ok with that, as long as we agree with respect to what "passing" means.

We should define passing as:

"All green, no yellow, and no red".

Anything other than "passing" is failing.

We are not planning on allowing for folks to say "we don't want to demonstrate interop on that feature, so its ok that we are "yelllow" for it...".